assets/xml/objects/object_kitan.xml

```diff
@@ -1,8 +1,8 @@
 ﻿<Root>
     <File Name="object_kitan" Segment="6">
```

Comment:
Nit, but can you put a comment above the `<File>` node to describe what's in this file?